[rospy.client][INFO] 2020-11-09 10:50:14,437: init_node, name[/train_3969_1604919014432], pid[3969]
[xmlrpc][INFO] 2020-11-09 10:50:14,438: XML-RPC server binding to 0.0.0.0:0
[xmlrpc][INFO] 2020-11-09 10:50:14,439: Started XML-RPC server [http://luigi:46173/]
[rospy.init][INFO] 2020-11-09 10:50:14,439: ROS Slave URI: [http://luigi:46173/]
[rospy.impl.masterslave][INFO] 2020-11-09 10:50:14,439: _ready: http://luigi:46173/
[rospy.registration][INFO] 2020-11-09 10:50:14,440: Registering with master node http://192.168.0.201:11311
[xmlrpc][INFO] 2020-11-09 10:50:14,441: xml rpc node: starting XML-RPC server
[rospy.init][INFO] 2020-11-09 10:50:14,539: registered with master
[rospy.rosout][INFO] 2020-11-09 10:50:14,540: initializing /rosout core topic
[rospy.rosout][INFO] 2020-11-09 10:50:14,545: connected to core topic /rosout
[rospy.simtime][INFO] 2020-11-09 10:50:14,546: /use_sim_time is not set, will not subscribe to simulated time [/clock] topic
[rospy.internal][INFO] 2020-11-09 10:50:14,569: topic[/bev/flow_image] adding connection to [http://192.168.0.201:43785/], count 0
[rospy.internal][INFO] 2020-11-09 10:50:14,571: topic[/odom] adding connection to [http://192.168.0.201:45777/], count 0
[rospy.internal][INFO] 2020-11-09 10:50:14,575: topic[/bev/occupancy_image] adding connection to [http://192.168.0.201:43785/], count 0
[rospy.internal][INFO] 2020-11-09 10:50:14,809: topic[/rosout] adding connection to [/rosout], count 0
[rospy.core][INFO] 2020-11-09 10:50:16,961: signal_shutdown [atexit]
[rospy.internal][INFO] 2020-11-09 10:50:16,965: topic[/rosout] removing connection to /rosout
[rospy.internal][INFO] 2020-11-09 10:50:16,966: topic[/bev/flow_image] removing connection to http://192.168.0.201:43785/
[rospy.internal][INFO] 2020-11-09 10:50:16,966: topic[/bev/occupancy_image] removing connection to http://192.168.0.201:43785/
[rospy.internal][INFO] 2020-11-09 10:50:16,966: topic[/odom] removing connection to http://192.168.0.201:45777/
[rospy.impl.masterslave][INFO] 2020-11-09 10:50:16,967: atexit
[rospy.internal][WARNING] 2020-11-09 10:50:16,968: Unknown error initiating TCP/IP socket to 192.168.0.201:34489 (http://192.168.0.201:43785/): Traceback (most recent call last):
  File "/opt/ros/noetic/lib/python3/dist-packages/rospy/impl/tcpros_base.py", line 562, in connect
    self.read_header()
  File "/opt/ros/noetic/lib/python3/dist-packages/rospy/impl/tcpros_base.py", line 657, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
AttributeError: 'NoneType' object has no attribute 'buff_size'

